configure

```diff
@@ -508,6 +523,10 @@ def configure_arm(o):
   o['variables']['arm_thumb'] = 0      # -marm
   o['variables']['arm_float_abi'] = arm_float_abi
 
+  if options.dest_os == 'android':
+    o['variables']['arm_fpu'] = 'vfpv3'
+    o['variables']['arm_version'] = '7'
```

Comment:
I added those macros just as plus to complete the list of ARM macros in compilers. Are not a must have to compile for Android but is good to have a complete list of macros.